crates/cmtraceopen-parser/tests/fixtures/sccm/server/site_database_export/v1/duplicate/export.json

```
@@ -0,0 +1 @@
+{"authorizationId":"first","authorizationId":"second"}
```

Biome (2.5.6)

[error] The key authorizationId was already declared.

(lint/suspicious/noDuplicateObjectKeys)